### ISSUE TITLE
Scope special series header styles for only neighborhoods landing pages and special projects template

### ISSUE THREAD

**Issue by joshdarby** [INN/umbrella-citylimits#168]
Helpscout thread: https://secure.helpscout.net/conversation/1218030674/5836/

Back in February when we did #161 and added the new special projects template header to the neighborhoods landing pages, I made the css for the special series header styling target all `.series-header-container` on any template, regardless of taxonomy and whatnot, when it was supposed to only be used on the new special projects landing page template we created and the existing neighborhoods landing page template.

https://github.com/INN/umbrella-citylimits/pull/161/files#diff-9e2b887c2360b2bc46aa8b1ac50c660c

tl;dr February @joshdarby made a boo-boo and it needs fixed.
